content/en/cloudcraft/components-common/icon.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "Icon Component"
+kind: documentation
+---
+
+## Overview
+
+The Icon component is one of the basic available components. Along with Images** and Blocks, it can be used to represent cloud components not yet available.
+
+{{< img src="cloudcraft/components-common/icon/component-icon.png" alt="Screenshot of a 3D representation of the icon component in Cloudcraft" responsive="true" style="width:60%;">}}
```

Comment:
```suggestion
{{< img src="cloudcraft/components-common/icon/component-icon.png" alt="Screenshot of a 3D representation of the icon component in Cloudcraft" responsive="true" style="width:60%;">}}
```

Not sure if this review is going through.. but this image is missing and there is no `icon` folder in this directory.